ambry-frontend/src/test/java/com.github.ambry.frontend/AmbrySecurityServiceTest.java

```diff
@@ -354,8 +354,13 @@ private void verifyHeadersForGetBlob(BlobProperties blobProperties, MockRestResp
     Assert.assertNull("TTL value should not be set", restResponseChannel.getHeader(RestUtils.Headers.TTL));
     Assert.assertNull("ServiceID value should not be set", restResponseChannel.getHeader(RestUtils.Headers.SERVICE_ID));
     Assert.assertNull("OwnerId value should not be set", restResponseChannel.getHeader(RestUtils.Headers.OWNER_ID));
-    Assert.assertNull("Content length value should not be set",
-        restResponseChannel.getHeader(RestUtils.Headers.CONTENT_LENGTH));
+    if (blobProperties.getBlobSize() < frontendConfig.chunkedGetResponseThresholdInBytes) {
```

Comment:
do test cases hit both if and else? don't you need to write a test case that checks both cases?